guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -116,13 +138,18 @@ The Discovery service is enabled by default on {ProjectServer}. However, the def
 
 image::PXE-mode.png[]
 
-To use {ProjectServer} to provide the Discovery image, install the `foreman-discovery-image` and `rubygem-smart_proxy_discovery` packages:
+To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
 
-----
-# yum install foreman-discovery-image rubygem-smart_proxy_discovery
-----
+ifeval::["{build}" == "foreman"]
+. `tfm-rubygem-foreman_discovery`
+endif::[]
+. `foreman-discovery-image`
+. `rubygem-smart_proxy_discovery`
```

Comment:
Not sure if `.` means a bullet, need to check.

Reply:
OK

Reply:
Note `tfm-` is a EL7 thing. Once we'll support EL8, it will change. For manual installs you can also use `foreman-plugin-discovery` which should be provided by the correct package. Similar we also have `foreman-proxy-plugin-discovery` instead of `rubygem-smart_proxy_discovery` (which will change to `tfm-rubygem-smart_proxy_discovery` in Foreman 1.24). Please do test this before putting in a manual though :)